### PR TITLE
Update build url

### PR DESCRIPTION
**Pull request by ghost**
- Add an additional global configuration item to indicate that we want to generate blue ocean URI's in comments posted back to Phabricator
- Update the logic that generates the build URL to use a different environment variable (exposed by blue ocean plugins) assuming we want to generate blue ocean URI's - otherwise leave it as it was
- Fixes: #269

### CODE REVIEW

**Comment by Sykomaniac** [comment]
image

**Comment by Sykomaniac** [comment]
Only a minor change but tested on our local Jenkins & phabricator instance and it works as expected:
ss

File: src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifierDescriptor.java [hunk 1]
```diff
@@ -103,4 +104,12 @@ public String getUberallsURL() {
     public void setUberallsURL(String value) {
         uberallsURL = value;
     }
+
+    public boolean getBlueOceanURI() {
```
Comment (ChaitanyaPramod) [inline src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifierDescriptor.java]: The methods and field make it appear like the property is a URI and not a boolean. Please change the property name

**Comment by Sykomaniac** [comment]
Pull request updated with new variable names - why is naming stuff always the hardest thing :)

**Pull request merged**